tensorflow/lite/examples/label_image/label_image.cc

```diff
@@ -45,6 +45,13 @@ limitations under the License.
 #include "tensorflow/lite/string_util.h"
 #include "tensorflow/lite/tools/evaluation/utils.h"
 
+#if defined(__ANDROID__)
+#include "tensorflow/lite/delegates/gpu/delegate.h"
+#if (defined(__arm__) || defined(__aarch64__))
+#include "tensorflow/lite/experimental/delegates/hexagon/hexagon_delegate.h"
```

Comment:
Is this needed? CreateHexagonDelegate() is defined in lite/tools/evaluation/utils.h

Reply:
@terryheo nope, they are not necessary. Both GPU and Hexagon headers are included in `lite/tools/evaluation/utils`. Changed per comment of @impjdi.

Reply:
gpu/delegate.h is needed for some definition such as TFLITE_GPU_INFERENCE_PREFERENCE_SUSTAINED_SPEED. But I don't think we need to include hexagon_delegate.h here if it's not needed.

Reply:
Thanks explanation. I'll remove it. BTW, how can I verify if some headers are needed?